### PR TITLE
Add translations for statements in 10 languages

### PR DESCRIPTION
**Pull request by dankim444**
## Description
This PR translates email_statements.csv, news_statements_amir.csv, and observable_gpt4o.csv in 9 new languages, contributing 27 new statement files in the raw_statements folder. This facilitates language support for statements as described in issue 145 of the commonsense-platform repository. 

## Changes
- Added new translated statements in the following languages: Arabic, Bengali, Spanish, French, Hindi, Japanese, Portuguese, Russian, and Chinese
- Updated the elicitation field to 'microsoft azure text translation' or 'amazon translate', depending on which tool was used.
- Updated the committer field to 'Dan'

## Additional Notes
Two programs have also been added to the .scripts folder: translate_statements_azure.py and translate_statements_aws. The translate_statements_azure.py script was used to translate news_statements_amir.csv and observable_gpt4o.csv, while translate_statements_aws.py was used to translate email_statements.csv.

### CODE REVIEW

**Review by amirrr — APPROVED** [review]
Nice job, this is very cool. I would like it if you could expose the loops you wrote for the translations as a function so we may use them in a utility module in the repo for automated translation or etc. 

Looking forward to seeing it working in conjunction with platform together.

**Comment by dankim444** [comment]
All duplicates have been removed from all the translation files, and the structure across all the files in the raw_statements directory remains consistent. I added two new scripts: show_groups_of_duplicates.py and remove_duplicates.py. show_group_of_duplicates.py extracts all the duplicates into separate csv files for processing purposes in remove_duplicates.py. remove_duplicates.py handles the actual removal and runs a test at the end to ensure the number of lines is the same across all the files.

@amirrr Could I get a final review on this before I merge?

File: .scripts/remove_duplicates.py
Comment (amirrr) [inline .scripts/remove_duplicates.py:16]: This is fine but let's keep track of our languages and I prefer a f string with list of languages for things like this
```python
f"raw_statements/observable_gpt4o_{language}.csv"
```

**Review by amirrr — APPROVED** [review]
This is good enough if there are no more duplicates. Create a new PR for workflow that checks for statements + translation. Create an issue with what you think is the best way to do this flow

**Pull request merged**
